_component_design/key-value-pairs.md

```diff
@@ -4,7 +4,7 @@ parent: Content Areas
 layout: component
 category: Components
 usage: >
-   Key-value pairs display data (the value) that corresponds to a label (the key). Color and alignment of key-value pairs allow the user to scan the page quickly for relevant data.
+   Key-value pairs display a label (the key) with the corresponding to data (the value). Color and alignment of key-value pairs allow the user to scan the page quickly for relevant data.
```

Comment:
Remove "to" in the first sentence.

s/ with the corresponding to data/ with the corresponding data